html/analysis/single_variable/differentiation.html

```diff
@@ -13,7 +13,16 @@
             <div class="definition" id="definition-derivative">
                 <div class="title">derivative</div>
                 <div class="content">
-                    Let `f` be a function and let `a` be a point in its domain, then the derivative of f at a is defined as: `lim_(h->0)(f(a+h)-f(a))/(h)`, when it exists and is denoted by `f^'(a)`
+                    Let `f` be a function and let `a` be a point in its domain, then the derivative of f at a is defined as: `lim_(h->0)(f(a+h)-f(a))/(h)`, when it exists and is denoted by `f^'(a)` or `(d)/(dx)f(a)`
+                </div>
+            </div>
+        </div>
+
+        <div class="thin-wrapper">
+            <div class="definition" id="definition-derivative">
+                <div class="title">partial derivative</div>
+                <div class="content">
+                    Let `U` be an open subset of `RR^n`, and let `f:U -> RR` be a function, then the partial derivative with respect to the `i`-th variable, `x_i`, of `f` at `a=(a_1,...,a_n) in U` is defined as: `lim_(h->0)(f(a_1,...,a_i + h,...,a_n)-f(a_1,...,a_i,...,a_n))/(h)` when it exists, and is denoted by `(del)/(del x_i) f(a)`
                 </div>
             </div>
```

Comment:
This definition looks good. But I think we create analysis/multi_variable/differentiation.html and put it there